ui/app/adapters/azure/config.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import ApplicationAdapter from '../application';
+import { encodePath } from 'vault/utils/path-encoding-helpers';
+
+export default class AzureConfig extends ApplicationAdapter {
+  namespace = 'v1';
+
+  queryRecord(store, type, query) {
+    const { backend } = query;
+    return this.ajax(`${this.buildURL()}/${encodePath(backend)}/config`, 'GET').then((resp) => {
+      return {
+        ...resp,
+        id: backend,
+        backend,
+      };
+    });
+  }
+
+  createOrUpdate(store, type, snapshot) {
+    const serializer = store.serializerFor(type.modelName);
+    const data = serializer.serialize(snapshot);
+    const backend = snapshot.record.backend;
+    return this.ajax(`${this.buildURL()}/${backend}/config`, 'POST', { data }).then((resp) => {
```

Comment:
is there a reason we're not encoding `backend` here? I wonder if it's be useful to pull the url logic would into a separate method so it's consistent. 

```js
_url(backend) {
return `${this.buildURL()}/${encodePath(backend)}/config`
}